recipes/officetopdf/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set name = "OfficeToPDF" %}
+{% set version = "1.8.22.0" %}
+{% set build = 0 %}
+{% set hash = "731caa224f101fc8bc292b90551d949cf4f35ea6afa10d890da53da0affe7dec" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/cognidox/{{ name }}/releases/download/v{{ version }}/{{ name }}.exe
+  sha256: {{ hash }}
+
+build:
+  skip: True  # [not win]
+  number: {{ build }}
+  script: "copy {{ name }}.exe %LIBRARY_BIN%"
```

Comment:
erm... could we compile this instead?  I'm not very keen on repacking exes

Reply:
good suggestion, I'll try to achieve that!